managed/services/supervisord/supervisord.go

```diff
@@ -491,6 +493,16 @@ func addAlertManagerParams(alertManagerURL string, templateParams map[string]int
 	return nil
 }
 
+// addPostgresParams adds pmm-server postgres database params to template config for grafana.
+func addPostgresParams(templateParams map[string]interface{}) {
+	templateParams["PostgresAddr"] = getValueFromENV("POSTGRES_ADDR", "127.0.0.1:5432")
+	templateParams["PostgresDBName"] = getValueFromENV("POSTGRES_DBNAME", "pmm-managed")
+	templateParams["PostgresDBUsername"] = getValueFromENV("POSTGRES_USERNAME", "pmm-managed")
+	templateParams["PostgresDBPassword"] = getValueFromENV("POSTGRES_DBPASSWORD", "pmm-managed")
+	templateParams["PostgresSSLKeyPath"] = getValueFromENV("POSTGRES_SSL_KEY_PATH", "")
+	templateParams["PostgresSSLCertPath"] = getValueFromENV("POSTGRES_SSL_CERT_PATH", "")
```

Comment:
I thought that we discussed getting these values from the kingpin.

Reply:
supervisord with grafana params from kingpin done